src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -115,6 +115,9 @@
         "text": {
             "defaultResponsePolicy": ["Always", "Conditional", "Never"]
         },
+        "coreSpecification" : "https://csa-iot.org/wp-content/uploads/2022/11/22-27349-001_Matter-1.0-Core-Specification.pdf",
```

Comment:
Why 1.0 and not 1.1?

In general, it seems like this will become out of date quickly...  can we point to something that actually lists all the versions of the specs?

Reply:
This spec was available publicly so I used this but we can point to any url here. If you have any suggestions for the links then I can just add those here instead.

Reply:
Since it is coming from the SDK. It might be better to point to the version that the SDK is currently implementing. If a user is still on an older sdk then they get their reference to spec they are on. That way they are not misaligned.

Reply:
That would make sense, except it would quickly get out of date, as I said.  The SDK is not implementing 1.0 right now, for sure.

Reply:
Anyway I can get public versions of the current Matter specification?

Reply:
@brdandu I don't know offhand.  That might be a question for @robszewczyk (ping him on Slack)?